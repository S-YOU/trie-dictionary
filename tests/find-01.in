find hello
insert hahahahaha
find hello
insert heloworld
find hello
insert ohmymistakeitshouldvebeenhello
find hello
insert imeanthello
find hello
insert helloiswhatimeant
find hello
