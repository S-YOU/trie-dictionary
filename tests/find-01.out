NO
word number: 0
NO
word number: 1
NO
word number: 2
NO
word number: 3
NO
word number: 4
YES
